udata_front/theme/gouvfr/templates/dataset/display.html

```diff
@@ -243,7 +243,11 @@ <h2 class="subtitle">{{ _('Temporality') }}</h2>
                                 {% endif %}
                                 <dt class="fr-text--sm fr-my-0 text-mention-grey">{{ _('Creation date') }}</dt>
                                 <dd class="fr-text--sm fr-ml-0 fr-mt-0 fr-mb-2w">
-                                    {{ dataset.created_at|dateformat(format='long') }}
+                                    {% if dataset.harvest and dataset.harvest.created_at %}
+                                        {{ [dataset.created_at, dataset.harvest.created_at]|max|dateformat(format='long') }}
```

Comment:
I don't think it should be the max for created_at. More like take the harvest date if set, else the internal one (as defined in https://github.com/etalab/data.gouv.fr/issues/1006)